clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -159,11 +155,7 @@ static bool IsSyclMathFunc(unsigned BuiltinID) {
   case Builtin::BItrunc:
   case Builtin::BI__builtin_trunc:
   case Builtin::BIceilf:
-  case Builtin::BI__builtin_ceilf:
-  case Builtin::BIcopysignf:
```

Comment:
Please, remove line 161 instead of this line.

Reply:
Submitted a fix. Thanks.